### DISCUSSION TITLE
Not able to mint the nft

### DISCUSSION THREAD

**Discussion by ARNAV1233** [Cyfrin/foundry-full-course-cu#170] [Q&A]
It is showing that the network is busy and not able to estimate the gas fee

**Comment by ChefAharoni** [comment]
For which challenge?
Have you entered the correct answer & your twitter handle with '@'?

**Comment by RichardPalladino** [comment]
> For which challenge? Have you entered the correct answer & your twitter handle with '@'?

I believe you can enter the correct answer either with or without the '@' symbol.  (I've tried both ways.)

As you hinted at, this sounds to be an issue with providing the incorrect answer.

**Comment by cromewar** [comment]
Please which lesson are you referring to?

**Comment by PatrickAlphaC** [comment]
Hello! Thanks for making this issue. 

Can you:
Could you please remember to follow this section to format your questions? https://youtu.be/umepbfKp5rI?t=22464
